frontend/src/components/OpeningsSearchTable/testData.ts

```diff
@@ -0,0 +1,55 @@
+import { ITableHeader } from "../../types/TableHeader";
+
+export const headers: ITableHeader[] = [
+  {
+    key: 'openingId',
+    header: 'Opening Id',
+    selected: true,
```

Comment:
Some linting errors here.  Mostly around unexpected commas.

Reply:
Got rid of the leading commas and did basic linting.